Formula/dxflib.rb

```diff
@@ -21,8 +22,54 @@ class Dxflib < Formula
     sha256 cellar: :any_skip_relocation, x86_64_linux: "6ab01712961c1a8561d2fc7926397b75169bb67ef89ebfadf4ee7ead4581d4c3"
   end
 
+  depends_on "qt" => :build
+
+  # Sample DXF file made available under GNU LGPL license.
+  # See https://people.math.sc.edu/Burkardt/data/dxf/dxf.html.
+  resource "testfile" do
+    url "https://people.math.sc.edu/Burkardt/data/dxf/cube.dxf"
+    sha256 "e5744edaa77d1612dec44d1a47adad4aad3d402dbf53ea2aff5a57c34ae9bafa"
+  end
+
   def install
-    system "./configure", "--prefix=#{prefix}"
-    system "make", "install"
+    system "qmake", "dxflib.pro"
+    system "make"
+    (include/"dxflib").install Dir["src/*"]
+    lib.install Dir["*.o", "*.a"]
```

Comment:
Why are we installing object files into `lib`? Is there no shared library?

Reply:
The previous version of the formula only installs `libdxflib.a` to `lib`, and I made these changes based on ArchLinux's PKGBUILD. There are no shared libraries with these particular steps, but I recall that using `make shared` might generate them. Should we do that instead?

Reply:
The build passed, so I guess no formulae in Homebrew/core was building a shared library. Still, it would probably be useful to build one if other formulae start using this as a dependency.

Also, installing `*.o` files into `lib` is a bit odd.

If the previous version installed a `.a` archive then we should keep it to avoid breaking anyone who relies on it being there.

Reply:
Does `qmake` really not define an `install` target? Odd.

Reply:
Yeah, for some reason there isn't an `install` target.